pom.xml

```diff
@@ -145,7 +153,7 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-gpg-plugin</artifactId>
-                        <version>1.5</version>
+                        <version>${maven-gpg-plugin.version}</version>
```

Comment:
Let's update some of the dependencies. For example, v1.5 is published in Feb 21, 2014 which is pretty old.